### ISSUE TITLE
Timecodes for "ML Zoomcamp Projects - Free parking slots detector by Timur Kamaliev"

### ISSUE THREAD

**Issue by alexeygrigorev** [DataTalksClub/machine-learning-zoomcamp#124]
Youtube video: https://www.youtube.com/watch?v=6AIc6aezibI

**Comment by dimzachar** [comment]
0:00:00 - Capstone project: predicting parking slot availability
0:02:16 - Organizing project structure using cookie cutter template.
0:05:00 - Organizing and splitting data for analysis.
0:07:41 - Using existing files for data distribution.
0:09:56 - Best practice: include variable types in code.
0:12:27 - EDA for parking occupancy deep learning project.
0:15:06 - Plotting feature and camera data distribution.
0:18:03 - Distribution of parking images by day.
0:20:58 - Parking data analyzed, images plotted.
0:24:19 - Creating average images for classification dataset.
0:27:05 - Expert: Preparing data for model training.
0:30:12 - Exploring two approaches to improve model
0:32:58 - Custom callback for tracking learning rate.
0:35:49 - Track experiments with customizable format & callbacks.
0:38:25 - Learning rate scheduler for improved accuracy.
0:41:23 - Tensorboard visualizes model performance and tuning.
0:45:07 - Validation data curves for inner size params.
0:48:56 - Experimenting with optimizers and error analysis.
0:52:25 - Using data augmentation to enhance images.
0:55:31 - Using special data loader for image augmentation.
0:58:58 - Implementing multiple filters, basic augmentations better.
1:01:53 - Testing model accuracy with augmentation techniques.
1:04:44 - Fine-tuning default model with augmented images.
1:07:35 - Training and validating a model for deployment.
1:10:58 - Separate config file for model training.
1:14:07 - Using data classes to read params.
1:17:08 - Converting Keras model to TensorFlow Lite.
1:20:15 - Deploying and testing a service using Docker.

**Comment by amitfrancis** [comment]
Updated timecodes. Thanks!
